haha, merging master-to-master with new fixes must be fun (#3)
dangreen/speed-up-with-browserslist: en/index.md

#### Module/nomodule

> With browsers' support of ES-modules, there was found a way to implement DSL on client side:

Reviewer: ```suggestion
With browsers' support of ES-modules, there is a way to implement DSL on client side:
```

</html>
```

In addition to scripts loading, there is support for [styles loading](https://github.com/TrigenSoftware/bdsl-webpack-plugin/tree/master/examples/postcss-preset-env). It is also possible to use this plugin on [server side](https://github.com/TrigenSoftware/bdsl-webpack-plugin/tree/master/examples/SsrBdslWebpackPlugin).

But, unfortunately, there are some nuances that you should know before starting to use bdsl-webpack-plugin: since scripts and styles loading is initialized by JavaScript, they are loaded asynchronously without render being blocked, and etc. For example, in case of the scripts — this means an inability to use `defer` attribute, and for the styles — the necessity to hide page content until styles are fully loaded. You can investigate how to get around these nuances, and other features of this plugin yourself, see [documentation](https://github.com/TrigenSoftware/bdsl-webpack-plugin/blob/master/README.md) and [usage examples](https://github.com/TrigenSoftware/bdsl-webpack-plugin#examples).

## Dependencies transpilation

> Following aforesaid part of the article, we've learned several ways of using Browserslist to reduce the size of the website's _own_ code, but the other part of the bundle is its dependencies. In web applications, the size of the dependencies in the final bundle can take up a significant part.

Reviewer: ```suggestion
Following the aforesaid part of the article, we've learned several ways of using Browserslist to reduce the size of the website's _own_ code, but the other part of the bundle is its dependencies. In web applications, the size of the dependencies in the final bundle can take up a significant part.
```
